cmd/openshift-install/create.go

```diff
@@ -267,6 +295,16 @@ func runTargetCmd(targets ...asset.WritableAsset) func(cmd *cobra.Command, args
 
 		err := runner(rootOpts.dir)
 		if err != nil {
+			if cmd.Name() == "cluster" {
```

Comment:
This either needs to be:
1. `if cmd.Name() == "cluster" || if cmd.Name() == "install-config" {` 
or
2. removed

I think removing it is fine. I don't think we need the check.

Reply:
Removed